svg/global_attributes.json

```diff
@@ -1,6 +1,50 @@
 {
   "svg": {
     "global_attributes": {
+      "autofocus": {
+        "__compat": {
+          "spec_url": "https://svgwg.org/svg2-draft/struct.html#autofocusattribute",
+          "tags": [
+            "web-features:autofocus"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "79"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": [
+              {
+                "version_added": "110"
+              },
+              {
+                "version_added": "78",
+                "partial_implementation": true,
+                "notes": "Only supported on <code>SVGGraphicsElement</code>."
```

Comment:
I'm not sure if this note doesn't need to be rephrased for this context.

Reply:
update to svg elements instead, searching via `SVGGraphicsElement` and `SVGGeometryElement` using https://github.com/mdn/content/blob/main/files/jsondata/InterfaceData.json